llvm/test/CodeGen/Hexagon/swp-ws-zero-cost.mir

```diff
@@ -0,0 +1,42 @@
+# REQUIRES: asserts
+# RUN: llc --march=hexagon %s -run-pass=pipeliner -debug-only=pipeliner \
+# RUN: -window-sched=force -filetype=null -verify-machineinstrs 2>&1 \
+# RUN: | FileCheck %s
+
+# CHECK-NOT: Can't find a valid II. Keep searching...
```

Comment:
A check-not check for a specific message is fragile and practically useless. This needs positive checks

Reply:
Updated

Reply:
CHECK-NEXT and -implicit-check-not are more reliable than putting a CHECK-NOT in one place